wolfcrypt/src/aes.c

```diff
@@ -1757,7 +1756,7 @@ static void wc_AesEncrypt(Aes* aes, const byte* inBlock, byte* outBlock)
 #ifdef DEBUG_WOLFSSL
     if (r > 7 || r == 0) {
         WOLFSSL_MSG("AesEncrypt encountered improper key, set it up");
-        return;  /* stop instead of seg-faulting, set up your keys! */
+        return KEYUSAGE_E;  /* stop instead of seg-faulting, set up your keys! */
```

Comment:
Does the comment `/* stop instead of seg-faulting, set up your keys! */` still apply?

Reply:
ah, right, n/a as I understand it -- removed.